src/ontology/uberon.Makefile

```diff
@@ -632,6 +632,21 @@ $(TMPDIR)/uberon-edit-plus-tax-equivs.owl: $(OWLSRC) $(TMPDIR)/external-disjoint
 	$(OWLTOOLS_CAT_DYNAMIC) $< $(TMPDIR)/external-disjoints.owl `ls $(BRIDGEDIR)/uberon-bridge-to-*.owl | grep -v emap.owl` --merge-support-ontologies -o -f ofn $@
 .PRECIOUS: $(TMPDIR)/uberon-edit-plus-tax-equivs.owl
 
+# uberon bridges to mba and dmba are now manually curated and generated in https://github.com/obophenotype/ABA_Uberon/tree/new_bridge
+# Note: the bridges are generated in new_bridge branch - this might change in the future, if it breaks here, please check ABA_Uberon repo to make sure that the new bridges are appropriately linked
+
+UBERON_BRIDGE_MBA = "https://raw.githubusercontent.com/obophenotype/ABA_Uberon/new_bridge/src/ontology/new-bridges/new-uberon-bridge-to-mba.owl"
```

Comment:
Risky to use a link to a branch rather than `main`..

Reply:
yes, I really disliked this when writing, but we don't have a choice atm, main is used as a comparison branch etc. 
For now if you don't mind, I'd rather leave it this way, but I think I'd want ot move the whole template building to bridge (so the tsv file and all) to uberon.

Reply:
Ok I would create an issue on uberon and assign it to yourself to remember this issue in the future, to prevent it going stale.